app/routes/talks.tsx

```diff
@@ -87,15 +97,182 @@ export const loader: LoaderFunction = async ({request}) => {
     checkValue: (value: unknown) => Array.isArray(value),
   })
 
-  const data: LoaderData = {talks}
+  const tags = Array.from(new Set(talks.flatMap(x => x.tags))).sort()
+
+  const data: LoaderData = {talks, tags}
   return json(data)
 }
 
+function Card({
+  tags,
+  title,
+  deliveries,
+  descriptionHTML,
+  resourceHTMLs,
+}: LoaderData['talks'][0]) {
+  const latestDate = deliveries
+    .filter(x => x.date)
+    .map(x => new Date(x.date as string))
+    .sort((l, r) => r.getTime() - l.getTime())[0] as Date
+
+  const isInFuture = latestDate.getTime() > Date.now()
+
+  // `software development` makes a terrible tag to render in this position
+  const tag = tags.filter(x => x.length <= 10)[0]
+
+  return (
+    <div className="block flex flex-col p-16 pr-24 w-full h-full bg-gray-100 dark:bg-gray-800 rounded-lg">
+      <div className="flex flex-none justify-between mb-8">
+        <div className="inline-flex items-baseline">
+          {isInFuture ? (
+            <div className="block flex-none w-3 h-3 bg-green-600 rounded-full" />
+          ) : (
+            <div className="block flex-none w-3 h-3 bg-gray-400 dark:bg-gray-600 rounded-full" />
+          )}
+          <H6 as="p" className="pl-4">
+            {formatDate(latestDate, 'PPP')}
+          </H6>
+        </div>
+
+        <div className="flex space-x-2">
+          {tag ? (
+            <div className="inline-block -mr-8 -mt-4 mb-4 px-8 py-4 text-black dark:text-white text-lg dark:bg-gray-600 bg-white rounded-full">
+              {tag}
+            </div>
+          ) : null}
+        </div>
+      </div>
+
+      <H3 as="div" className="flex flex-none items-end mb-4 h-48">
```

Comment:
Let's add `import slugify from '@sindresorhus/slugify'` to the top and then above: `const slug = slugify(title)` and then:

```suggestion
      <H3 as="a" href={`#${slug}`} id={slug} className="flex flex-none items-end mb-4 h-48">
```

Or something like that. So we can link directly to one of the talks. I don't know whether it would be possible to somehow highlight the one that's selected as well, but that would probably be good too. I link people to my talks pretty often.

Reply:
The highlight isn't an issue, but the slug is. Talk titles agent unique due to the TBA's.

Reply:
Oh, sweet. This will work.

```js
import {slugifyWithCounter} from '@sindresorhus/slugify';

const slugify = slugifyWithCounter();

slugify('foo bar');
//=> 'foo-bar'

slugify('foo bar');
//=> 'foo-bar-2'

slugify.reset();

slugify('foo bar');
//=> 'foo-bar'
```